### DISCUSSION TITLE
[Feat]: 能否有一个定时启动的功能？

### DISCUSSION THREAD

**Discussion by i-cecreamer** [moesnow/March7thAssistant#39] [Q&A] [answered]
### 请确保您已阅读以上注意事项，并勾选下方的确认框。

- [X] 我确定这是一个从未被提出和实现过的功能。
- [X] 我已经在 [Issue Tracker](https://github.com/moesnow/March7thAssistant/issues) 以及 [Pull Request](https://github.com/moesnow/March7thAssistant/pulls) 中找过我要提出的功能，没有找到相同功能的ISSUE和PR。
- [X] 我已知晓并同意，此处仅用于建议新功能。若这个 Issue 是关于其他非主题的问题，则我的 Issue 可能会被无条件自动关闭/锁定。
- [X] 我已知晓并同意，如果我不按照模板提供的指示进行填写，则我的 Issue 可能会被无条件删除。

### 需要添加的功能内容

因为我不是整天开着电脑挂着程序后台的，我在睡前是习惯用按键助手模拟4点自动鼠标左键点击完整运行，但是每次需要计算距离4点还要多久点击鼠标左键，希望有一个能让程序在后台设定时间自动开启的功能，或者已经有了，但是我没发现？

**Comment by moesnow** [comment, marked as answer]
两个方法任选其一
1. 设置中的“[任务完成后](https://moesnow.github.io/March7thAssistant/#/assets/docs/Tutorial?id=%e4%bb%bb%e5%8a%a1%e5%ae%8c%e6%88%90%e5%90%8e)”选项，修改为“循环”
2. 使用Windows自带的“任务计划程序”，设定4点启动“[March7th Assistant.exe](https://github.com/moesnow/March7thAssistant#下载安装)”（注意不是“March7th Launcher.exe”）

**Reply by ttupwz** [reply]
你好，我是一个小白，我现在使用的就是第二个方案，但是我想要实现4点启动8点关闭，我在八点关闭里设置taskkill -f -im March7th Assistant.exe ,但是march7th assistant依然在运行，类似的starrail.exe都能关闭，怎么才能关闭march7th让它不再自检启动starrail呢，

**Reply by moesnow** [reply]
尝试增加引号 `taskkill -f -im "March7th Assistant.exe"`
